license.gradle

```diff
@@ -0,0 +1,196 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+apply plugin: "com.github.jk1.dependency-license-report"
+
+tasks.register('compareLicenseReport') {
+  doLast {
+    File licenseFile = new File("${projectDir}/build/reports/dependency-license/LICENSE")
+    File currentLicenseFile = new File("${projectDir}/LICENSE")
+    if (licenseFile.text != currentLicenseFile.text) {
+      throw new RuntimeException("License file has changed");
```

Comment:
Maybe mention about what to do next, like Run `updateLicenseReport` with a complete gradle command. 

Similar to how RevAPI or spotless prompts when fails about the next step.

Reply:
Thanks, I added this.